Source/Common/Util.cpp

```diff
@@ -107,6 +107,7 @@ bool pjutil::TerminatedExistingExe()
                     if (TerminateProcess(hHandle, 0))
                     {
                         bTerminated = true;
+                        WaitForSingleObject(hHandle, 30 * 1000);
```

Comment:
Yep, but he likes figuring out things for the compiler so I just felt like being an ass.